models/library.go

```diff
@@ -53,19 +63,56 @@ type Library struct {
 	// The Path to the library in the container image. Empty string when Lockfile scan.
 	// This field is used to convert the result JSON of a `trivy image` using trivy-to-vuls.
 	FilePath string
+	Digest   string
+}
+
+func trivyInit() error {
+	if err := tlog.InitLogger(config.Conf.Debug, config.Conf.Quiet); err != nil {
```

Comment:
It looks good to execute it here.
https://github.com/future-architect/vuls/blob/74aa70a24ac7723f68d37c6f39fcb13285a1e3b9/detector/library.go#L28